### DISCUSSION TITLE
populate devices into netbox

### DISCUSSION THREAD

**Discussion by selvigp** [netbox-community/netbox#15233] [Q&A]
I have created Python scripts to populate devices into NetBox, and I ran my scripts in NetBox WebUI (under customization). But I am getting insufficient write permission to the media root error message

<img width="270" alt="netbox" src="https://github.com/netbox-community/netbox/assets/54948133/456dd6c8-f06d-4f70-9ec3-4b5347af94f3">

**Comment by deku-m** [comment]
Looks like permissions not set.
Try:
sudo chown --recursive netbox /opt/netbox/netbox/media/
sudo chown --recursive netbox /opt/netbox/netbox/reports/
sudo chown --recursive netbox /opt/netbox/netbox/scripts/

https://docs.netbox.dev/en/stable/installation/3-netbox/#create-the-netbox-system-user

For docker in the compose file:
    - netbox-media-files:/opt/netbox/netbox/media:rw
    - netbox-reports-files:/opt/netbox/netbox/reports:rw
    - netbox-scripts-files:/opt/netbox/netbox/scripts:rw
